packages/menu/src/SubMenu/SubMenu.tsx

```diff
@@ -223,29 +223,29 @@ export const SubMenu = InferredPolymorphic<SubMenuProps, 'button'>(
           )}
         >
           {content}
-          <IconButton
-            data-testid="lg-sub-menu-icon-button"
-            darkMode={!darkMode}
-            ref={setIconButtonElement}
-            aria-label={open ? 'Close Sub-menu' : 'Open Sub-menu'}
-            className={cx(
-              iconButtonClassName,
-              iconButtonStyle,
-              iconButtonThemeStyle[theme],
-              iconButtonFocusedThemeStyle[theme],
-              {
-                [openIconButtonStyle[theme]]: open,
-              },
-            )}
-            onClick={handleChevronClick}
-          >
-            <ChevronIcon
-              role="presentation"
-              className={cx(chevronClassName, chevronIconStyles)}
-              size={14}
-            />
-          </IconButton>
         </Component>
+        <IconButton
+          data-testid="lg-sub-menu-icon-button"
+          darkMode={!darkMode}
+          aria-label={open ? 'Close Sub-menu' : 'Open Sub-menu'}
+          className={cx(
+            iconButtonClassName,
+            iconButtonStyle,
+            iconButtonThemeStyle[theme],
+            iconButtonFocusedThemeStyle[theme],
+            iconButtonPosition[size],
+            {
+              [openIconButtonStyle[theme]]: open,
+            },
+          )}
+          onClick={handleChevronClick}
```

Comment:
need to stop prop here, because this is firing the onClick for the submenu (check .design example)

Reply:
I think this has been fixed